syft/generic/pointers/pointer_tensor.py

```diff
@@ -180,10 +180,10 @@ def get_class_attributes(self):
     @staticmethod
     def create_pointer(
         tensor,
-        location: AbstractWorker = None,
+        location: Union[AbstractWorker, str] = None,
         id_at_location: (str or int) = None,
         register: bool = False,
-        owner: AbstractWorker = None,
+        owner: Union[AbstractWorker, str] = None,
```

Comment:
Would be nice if we can keep the types of these parameters simple. Does this help with another part of the PR? Feel like I might have missed something.

Reply:
I just felt guilty and wanted to pay tribute to the 2 `owner = tensor.owner.get_worker(owner)` below :/

Reply:
In that case the `owner` param is a string?

Reply:
Yes it can be!